robber/matchers/exception.py

```diff
@@ -0,0 +1,33 @@
+from robber import expect
+from robber.matchers.base import Base
+
+
+class ExceptionMatcher(Base):
+    """
+    expect(lambda: call_something(with_some_params)).to.throw(any_exception)
+    """
+    @property
+    def _raised(self):
+        raised = None
+
+        if not callable(self.actual):
+            raise TypeError('{actual} is not callable'.format(actual=self.actual))
+
+        try:
+            self.actual()
+        except BaseException as raised:
+            return raised
+
+    def matches(self):
+        return isinstance(self._raised, self.expected)
```

Comment:
Yup, but I'm still thinking about supporting chaining inside the matcher object :D. It should solve the above use case, and also introduce more sexier syntax:
```python
expect(callable).to.change(this_var).from(1).to(2)
expect(callable).to.increase(this_var).by(1)
```
:D.